package.json

```diff
@@ -33,11 +33,11 @@
     "@capacitor/cli": "^2.4.7",
     "@intlify/vue-i18n-loader": "^2.1.0",
     "@types/jest": "^24.0.19",
-    "@typescript-eslint/eslint-plugin": "^5.4.0",
-    "@typescript-eslint/parser": "^5.4.0",
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
```

Comment:
Commerce Hub working without this change
https://github.com/hotwax/commerce-hub/blob/main/package.json#L36
Any specific reason for introducing this change?

Reply:
Sir, I have taken reference from StackOverflow to resolve these build failures related to node's version 16. 
Reverted these changes from code after downgrading and checked after a fresh install.